.github/workflows/kind_host.yml

```diff
@@ -0,0 +1,157 @@
+name: Framework KinD hosted mode Workflow
+
+on:
+  workflow_call:
+
+env:
+  RELEASE_BRANCH: ${{ github.event.pull_request.base.ref || github.ref_name }}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  tests:
+    name: Hosted mode Tests # Part of the check name, be careful when changing.
+
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: localhost:5000
+    strategy:
+      # Don't skip all tests if one fails - others may still have useful info
+      fail-fast: false
+      matrix:
+        # Run tests against a new version of kubernetes
+        # KinD tags are listed at https://hub.docker.com/r/kindest/node/tags
+        kind:
+          - 'latest'
+        deployOnHub:
+          - 'false'
```

Comment:
This isn't applicable to this setup so could you please remove this from the matrix?